src/tailwind/index.cjs

```diff
@@ -107,6 +107,9 @@ function mergeArray(objValue, srcValue) {
 
 const themeColors = {
   'banano-base': tailwindColors.blue,
+  'banano-text-foreground': tailwindColors.gray['900'],
+  'banano-text-muted': tailwindColors.gray['500'],
+  'banano-bg': tailwindColors.white,
```

Comment:
yes, you could technically use `text-banano-bg`. Not sure if I understood correctly, are you saying to use `text-banano-muted` instead of  `text-banano-text-muted`, and `bg-banano` instead if `bg-banano-bg`? It's possible, but I would still be able to use something like `text-banano` and that would give me the color of the bg, which might not be intuitive I think. This colors are not really meant to be used in the target application directly, more internally and to customize what banano offers, so I think it would be better for the names to be obvious in the tailwind config theme, where they might be overriden. Does it make sense?

Reply:
Yes, that is what I was saying. Okay I understand, It makes sense to me if It is only to intern use and It is more explicit to the user what color is customizing.